packages/alfa-rules/src/sia-r83/rule.ts

```diff
@@ -104,6 +111,36 @@ export default Rule.Atomic.of<Page, Text>({
   },
 });
 
+function hasDisplaySize(
+  valueOrPredicate: number | Predicate<number>
+): Predicate<Element> {
+  const predicate =
+    typeof valueOrPredicate === "function"
+      ? valueOrPredicate
+      : (size: number) => valueOrPredicate === size;
+
+  return (element) => {
+    let displaySize = 0;
+    for (const size of element.attribute("size")) {
+      const valueMenuSize = parseInt(size.value, 10);
+      if (
+        valueMenuSize === valueMenuSize &&
+        valueMenuSize === (valueMenuSize | 0)
+      ) {
+        displaySize = valueMenuSize;
+      }
+    }
+
+    if (element.attribute("multiple").isSome()) {
+      displaySize = 4;
+    }
+
+    displaySize = 1;
```

Comment:
Now, that one is always executed (there is no `return` in the test) and erase the value 😓 
Easiest way would just be to initialise `displaySize` to 1 instead of 0 and remove this line.